core/src/main/scala/tech/beshu/ror/utils/SSLCertHelper.scala

```diff
@@ -51,6 +60,29 @@ object SSLCertHelper extends Logging {
     }
   }
 
+  private def loadPrivateKey(file: File): IO[PrivateKey] = {
```

Comment:
can we move the private method below the method where it's used for the first time?